manticore/ethereum/solidity.py

```diff
@@ -139,7 +139,7 @@ def get_source_for(self, asm_offset, runtime=True):
             return ''
 
         output = ''
-        nl = self.source_code[:beg].count('\n')
+        nl = self.source_code[:beg].count('\n') + 1
```

Comment:
this will handle the case where the file does not end in a newline, but if it ends in a newline it'll overcount by 1 right?

Reply:
Hmm I believe not. The `self.source_code` is a string containing all the source and `beg` is the index where given code starts.

So for a code like this:
```
contract X {
    function run(uint arg0, uint arg1) public returns (uint) {
        return arg0 - arg1;
    }
}
```
we will count two lines and later we will create a string like `2   return arg0 - arg1;`.